src/modules/dashboard/components/ActionsPage/ActionPageMotionContent/ActionPageMotionContent.tsx

```diff
@@ -0,0 +1,161 @@
+  createdAt: number;
+  currentUserName?: string | null;
+  roleMessageDescriptorId?: string | null;
+}
+
+const ActionPageMotionContent = ({
+  colony,
+  colonyAction: { annotationHash, events = [], actionType, rootHash },
+  colonyAction,
+  isDecision,
+  initiator,
+  walletAddress,
+  actionAndEventValues,
+  roleTitle,
+  transactionHash,
+  userHasProfile,
+  motionId,
+  bottomElementRef,
+  createdAt,
+  currentUserName,
+  roleMessageDescriptorId,
+}: Props) => {
+  const { data: objectionAnnotation } = useMotionObjectionAnnotationQuery({
+    variables: {
+      motionId,
+      colonyAddress: colony.colonyAddress,
+    },
+    fetchPolicy: 'network-only',
+  });
+
+  const { data: motionEventsData } = useEventsForMotionQuery({
+    variables: { colonyAddress: colony.colonyAddress, motionId },
+    fetchPolicy: 'network-only',
+  });
+
+  const { data: motionsSystemMessagesData } = useMotionsSystemMessagesQuery({
+    variables: {
+      motionId,
+      colonyAddress: colony.colonyAddress,
+    },
+    fetchPolicy: 'network-only',
+  });
+
+  const objectionAnnotationUser = useUser(
+    objectionAnnotation?.motionObjectionAnnotation?.address || '',
+  );
+
+  return (
+    <>
+      {isDecision ? (
+        <ActionPageDecisionWithIPFS
+          colony={colony}
+          user={initiator}
+          username={currentUserName || ''}
+          walletAddress={walletAddress}
+          hash={annotationHash || ''}
+          createdAt={createdAt}
+        />
+      ) : (
+        <h1 className={styles.heading} data-test="actionHeading">
+          <FormattedMessage
+            id={roleMessageDescriptorId || 'motion.title'}
+            values={{
+              ...actionAndEventValues,
+              fromDomainName: actionAndEventValues.fromDomain?.name,
+              toDomainName: actionAndEventValues.toDomain?.name,
+              roles: roleTitle,
+            }}
+          />
+        </h1>
+      )}
+
+      {isDecision
+        ? objectionAnnotation?.motionObjectionAnnotation?.metadata && (
+            <div className={styles.annotation}>
+              <ActionPageDecisionWithIPFS
+                colony={colony}
+                user={objectionAnnotationUser}
+                username={currentUserName || ''}
+                walletAddress={walletAddress}
+                hash={objectionAnnotation?.motionObjectionAnnotation?.metadata}
+                isObjection
+                createdAt={createdAt}
+              />
+            </div>
+          )
+        : objectionAnnotation?.motionObjectionAnnotation?.metadata && (
+            <div className={styles.annotation}>
+              <ActionsPageFeedItemWithIPFS
+                colony={colony}
+                user={objectionAnnotationUser}
+                annotation
+                hash={objectionAnnotation.motionObjectionAnnotation.metadata}
+                appearance={{ theme: 'danger' }}
+              />
+            </div>
+          )}
+
+      <ActionsPageFeed
```

Comment:
I would prefer that ActionsPageFeed & CommentInput should stay in the DefaultMotion. There is no reason to take them out. It would be more consistent with DefaultAction structure as well... What do you think?

Reply:
@jakubcolony @chinins Sorry if this adds a little confusion, but the way I see it, is that all of the decision/motion content is handled in the new component. It's encapsulated. It will be easier to maintain. Which is how its been refactored.

Otherwise whats the responsibility of the new component? only handling the heading/title & objections? I think thats a bit messy. 
But If so, then the components name should be changed. 

I'll go along with the option that you decide........

Reply:
I'm leaning towards Dan's view that this new component handles all the content rather than just the heading but I'll wait to see if I can get another reviewer's opinion